Alpaca-Finance-Agents/AllocPoint-Change-Alert/src/agent.ts

```diff
@@ -0,0 +1,97 @@
+import {
+  Finding, 
+  HandleTransaction, 
+  TransactionEvent, 
+  FindingSeverity, 
+  FindingType,
+  LogDescription
+} from 'forta-agent';
+import { 
+  decodeParameters
+} from "forta-agent-tools";
+
+const POOL_OWNERS: string[] = [
+  "0xc48FE252Aa631017dF253578B1405ea399728A50", // MDEX - BscPool
+  "0xA1f482Dc58145Ba2210bC21878Ca34000E2e8fE4"  // PancakeSwap - Timelock
+]
+
+export const setFuncAbi: string = "function set(uint256 _pid, uint256 _allocPoint, bool _withUpdate)";
+export const setFuncSig: string = "set(uint256,uint256,bool)";
+
+const queueTxnAbi: string = "event QueueTransaction(bytes32 indexed txHash, address indexed target, uint value, string signature, bytes data, uint eta)";
+
+function isAddressRelevant(contractAddress: string, poolArray: string[]): boolean {
+  return poolArray.includes(contractAddress);
+}
+
+function containsFuncSig(log: LogDescription, functionSig: string): boolean {
+  return log.args["signature"] === functionSig;
+}
+
+const createFinding = (
+  poolId: number,
+  allocPoint: number,
+  withUpdate: boolean,
+  target: string | null
+  ): Finding => {
+  return Finding.fromObject({
+    name: "AllocPoint Change Event",
+    description: "Pool's alloc point queued for update.",
+    alertId: "ALPACA-6",
+    severity: FindingSeverity.Info,
+    type: FindingType.Info,
+    metadata:{
+      poolId: poolId.toString(),
+      allocPoint: allocPoint.toString(),
+      withUpdate: withUpdate.toString(),
+      target: target || "N/A" // target ARGUMENT COULD POTENTIALLY BE null FROM txEvent.to
+    },
+  })
+}
+
+export function provideHandleTransaction(
+  poolOwners: string[]
+): HandleTransaction {
+  return async (txEvent: TransactionEvent): Promise<Finding[]> => {
+    const findings: Finding[] = [];
+
+    findings.push(
+      ...txEvent.filterLog(queueTxnAbi)
+        .filter(log => isAddressRelevant(log.address, poolOwners))
```

Comment:
I would suggest to always use lowers everywhere to avoid possible errors if the `POOL_OWNERS` array change. Right now I guess everything works well because the array is using checksumed addresses, but if the client change that list and is not aware of this the agent will not work as expected.

So maybe is safer to add a `.map(addr => addr.toLowerCase())` in the end of L16, and also use `log.addres..toLowerCase()` in the highlighted line above